### PR TITLE
doc: add README for windows e2e

### PR DESCRIPTION
**Pull request by junjiezhang1997**
**What type of PR is this?**
/kind doc

**What this PR does / why we need it**:
Add README for Windows E2E.

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: e2e/windows/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- **pod-windows-template.yaml**. This pod yaml file is used to deploy a test pod to validate whether the node is already joined to the test cluster.
+- **scenarios/windows/property-windows-template.json**. It is the third and last part of template used to generate windows CustomData file and CSECmd file and it contains properties that are related to the test windows nodepool.
+- **upload-cse-logs.ps1**. This script is used to upload generated cse logs to the stroage account in the deployed windows vmss for further download.
+- **windows_vmss_template.json**. This json file is the basic template used to deploy a windows vmss. We add necessary properties to this file in `e2e-scenario.sh`.
+
+## Generate and Update current windows test images
+Currently, we use test images "/subscriptions/a15c116e-99e3-4c59-aebc-8f864929b4a0/resourceGroups/akswinvhdbuilderrg/providers/Microsoft.Compute/galleries/AKSWindows/images/windows-e2e-test-$WINDOWS_E2E_IMAGE/versions/2023.02.07/" ($WINDOWS_E2E_IMAGE could be 2019-containerd, 2022-containerd, 2022-containerd-gen2) as is referred by `IMAGE_REFERENCE` in `e2e-scenario.sh`.
```
Comment (wanqingfu) [inline e2e/windows/README.md:54]: I think the sub id should not be exposed here as it's a public repo.

File: e2e/windows/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- **upload-cse-logs.ps1**. This script is used to upload generated cse logs to the stroage account in the deployed windows vmss for further download.
+- **windows_vmss_template.json**. This json file is the basic template used to deploy a windows vmss. We add necessary properties to this file in `e2e-scenario.sh`.
+
+## Generate and Update current windows test images
+Currently, we use test images "/subscriptions/$SUBSCRIPTION_ID/resourceGroups/akswinvhdbuilderrg/providers/Microsoft.Compute/galleries/AKSWindows/images/windows-e2e-test-$WINDOWS_E2E_IMAGE/versions/2023.02.07/" ($WINDOWS_E2E_IMAGE could be 2019-containerd, 2022-containerd, 2022-containerd-gen2) as is referred by `IMAGE_REFERENCE` in `e2e-scenario.sh`.
+
+To generate new windows test images, we can use [`AKS Windows VHD Build - PR check-in gate`](https://msazure.visualstudio.com/CloudNativeCompute/_build?definitionId=182855&_a=summary). Before running the pipeline, we need to set the values of pipeline variables as follows:
```
Comment (wanqingfu) [inline e2e/windows/README.md:56]: should we include the pipeline link here as I think it's also sensitive info?

Comment (junjiezhang1997) [reply e2e/windows/README.md:56]: This pipeline could also be clicked in the following checks, so I think it's ok here.

File: e2e/windows/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# Introduction of Windows E2E
+## Summary 
+Windows E2E is used to validate the latest change of windows cse package automatically. We current support three versions of windows images: windows 2019 containerd, windows 2022 containerd gen1 and windows 2022 containerd gen2.
```
Comment (AbelHu) [inline e2e/windows/README.md:3]: current => currently?

**Review by AbelHu — APPROVED** [review]

**Pull request merged**